### DISCUSSION TITLE
Collapsed format not supported through jfrconv binary

### DISCUSSION THREAD

**Discussion by hridayahuja44** [async-profiler/async-profiler#983] [General]
I am currently using this command to generate the flame graphs where I have enabled CPU, Wall-clock with alloc and lock parameters along with jfrsync enabled for the profiling:
<img width="999" alt="Screenshot 2024-09-02 at 22 13 46" src="https://github.com/user-attachments/assets/f351df9e-5058-4166-b98e-9841f33ec7c4">

While profiling with jfrsync enabled I noticed that while converting the jfr format to the collapsed format, this being the command for the same:
<img width="353" alt="Screenshot 2024-09-02 at 22 04 05" src="https://github.com/user-attachments/assets/f815a54b-8c3c-4088-8800-c3d06b1a2a17">

**we encountered this error:**
<img width="1209" alt="Screenshot 2024-09-02 at 22 02 42" src="https://github.com/user-attachments/assets/8b220db0-1ce6-48b3-bde6-b44c640d1984">

**The error is pointing towards this file:** 
https://github.com/async-profiler/async-profiler/blob/d92b893826e4de284851cff64662ed58b171885e/src/converter/Main.java#L46
https://github.com/async-profiler/async-profiler/blob/d92b893826e4de284851cff64662ed58b171885e/src/converter/Main.java#L60

**This is the code snippet for the same:**

<img width="846" alt="Screenshot 2024-09-02 at 22 07 19" src="https://github.com/user-attachments/assets/6610e94d-4dcc-45df-9ce5-490373893c0b">
@apangin, could you please help resolve this issue, if there is some other collapsed format nomenclature being used you can let me know I will modify the command accordingly

**Comment by hridayahuja44** [comment]
@apangin any update on this issue?

**Reply by apangin** [reply]
The fix has been pushed and will be included in the next release.
In the meantime, you may use [Nightly builds](https://github.com/async-profiler/async-profiler/releases/tag/nightly).

**Comment by apangin** [comment]
Direct jfr->collapsed conversion is not currently supported. We will fix this shortly.

**Reply by hridayahuja44** [reply]
Okay thanks a lot @apangin, if it's possible could you make the fix as soon as possible cause a service we are trying to implement depends on this conversion from jfr -> collapsed

**Reply by hridayahuja44** [reply]
Hey @apangin, any update on the fix for this issue I have a service that I am planing to run and I am getting a bit blocked because of this. Could you please help fixing this issue

**Reply by apangin** [reply]
The problem is noted and put on our backlog. We take all our users' feedback carefully and prioritize our work accordingly. Demanding immediate resolution of your issues does not help to speed up the process; furthermore, it violates the [Code of Conduct](https://aws.github.io/code-of-conduct) adopted by async-profiler project. Thank you for your patience.

**Reply by hridayahuja44** [reply]
Okay understood, my apologies @apangin

**Comment by hridayahuja44** [comment]
Hey @apangin, any update on this issue

**Reply by apangin** [reply]
As I've mentioned [above](https://github.com/async-profiler/async-profiler/discussions/983#discussioncomment-10757060), the issue had been fixed. You may use new converter from the [nightly builds](https://github.com/async-profiler/async-profiler/releases/tag/nightly). Let me know if you have further questions.
